src/style.css

```
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+  
+
+  --page-color: #00acdc;
```

```suggestion
  margin: 0;
  
  --page-color: #00acdc;
```
No need in so many spaces